frontend/assets/stylesheets/components/_packages.scss

```diff
@@ -7,149 +7,178 @@
    - Main package panel used across the site
    ========================================================================== */
 
-.package-promo {
-    position: relative;
+// This is the height of the actions block that contains the price & call-to-action button
+$_package-promo__actions_height: 117px;
+
+@mixin package-promo-base() {
     border-top: 5px solid;
     background-color: $c-neutral7;
-}
+    padding: rem($gs-gutter / 4);
+
     .package-promo__header {
-        padding: rem($gs-gutter / 4);
+        margin-bottom: rem($gs-gutter / 2);
```

Comment:
You can remove the `rem()` function call here as they are no longer needed (we still have a few files with them in if you fancy doing some cleanup).